tests/test_markup/test_markup.py

```diff
@@ -30,7 +30,8 @@ def settings(app):
         # DeprecationWarning: The frontend.OptionParser class will be replaced
         # by a subclass of argparse.ArgumentParser in Docutils 0.21 or later.
         optparser = frontend.OptionParser(
-            components=(RstParser, HTMLWriter, LaTeXWriter), defaults=default_settings
+            components=(RstParser, HTMLWriter, LaTeXWriter),  # type: ignore[arg-type]
```

Comment:
what error is being suppressed here?
if missing generics, can we add the generics?

Reply:
```
tests\test_markup\test_markup.py:33:24: error: Argument "components" to "OptionParser" has incompatible type "tupl
e[type[Parser], type[HTMLWriter], type[LaTeXWriter]]"; expected "Iterable[Component]" 
```

Reply:
All of these are subclasses of `Component`. The error is probably because either types or instances are allowed -- do the stubs `frontend.OptionParser` reflect this? (I suspect not because it expects `Iterable[Component]`

Reply:
can it really be an instance or a class? that's chaotic.
i suspect then that the type stubs need an update and suppressing the lint is the way to go for now

Reply:
see https://github.com/python/typeshed/pull/12735

Reply:
Thanks!